js/TodoList.js

```diff
@@ -0,0 +1,84 @@
+import { getTodoItems, postTodoItem, deleteTodoItem } from "./Fetch.js";
+const PRIORITY = {
+  NONE: `<select class="chip select">
+  <option value="0" selected>순위</option>
+  <option value="1">1순위</option>
+  <option value="2">2순위</option>
+</select>`,
+  FIRST: `<span class="chip primary">1순위</span>
+`,
+  SECOND: `<span class="chip secondary">2순위</span>`,
+};
+const $todoList = document.querySelector(".todo-list");
+let USERID = "";
+export const initTodoList = () => {
+  addListeners();
+  const $newTodo = document.querySelector(".new-todo");
+  $newTodo.addEventListener("keyup", addItem);
+};
+
+export const renderTodoListOfUser = async ({ target }) => {
+  const classes = target.className.split(" ");
+  USERID = target.id;
+
+  if (!classes.includes("userBtn")) return false;
```

Comment:
이 부분은 ```split()```, ```includes()``` 메서드 대신에 아래와 같이 ```classList.contains()``` 메서드를 사용하시면 더 좋을거 같습니다!
```suggestion
  if (!target.classList.contains("userBtn")) return false;
```

Reply:
오 contains는 처음보는 메소드네요... 한 번에 해결할 수 있어서 좋네요!!